include/lsst/afw/detection/Footprint.h

```diff
@@ -291,7 +192,7 @@ class Footprint : public lsst::daf::base::Citizen,
      *
      */
     template <typename MaskPixelT>
-    void intersectMask(
+    std::shared_ptr<Footprint> intersectMask(
         image::Mask<MaskPixelT> const & mask,
         MaskPixelT bitmask=~0x0
     );
```

Comment:
There seems to be only one or two places that use this. I am removing it and will address those places in the fix-up ticket.